app/Server/src/SERVERAPI/Pages/RanchFields/CreateEdit.cshtml.cs

```diff
@@ -0,0 +1,221 @@
+        {
+            return await ProcessPost();
+        }
+
+        public async Task<IActionResult> OnPostEditAsync()
+        {
+            return await ProcessPost();
+        }
+
+        private async Task<IActionResult> ProcessPost()
+        {
+            if (ModelState.IsValid)
+            {
+                await _mediator.Send(Data);
+
+                if (IsModal)
+                {
+                    return this.RedirectToPageJson(nameof(Index));
+                }
+                return RedirectToPage(nameof(Index));
+            }
+            Data = await _mediator.Send(new LookupDataQuery { PopulatedData = Data });
+            return Page();
+        }
+
+        public class Query : IRequest<Command>
+        {
+            public int Id { get; set; }
+        }
+
+        public class LookupDataQuery : IRequest<Command>
+        {
+            public Command PopulatedData { get; set; }
+        }
+
+        [BindProperties]
+        public class Command : IRequest<MediatR.Unit>
+        {
+            public int Id { get; set; }
+            public string FieldName { get; set; }
+            //public Nutrients Nutrients { get; set; }
+
+            //public List<FieldCrop> Crops { get; set; }
+            //public SoilTest SoilTest { get; set; }
+
+            public string FieldArea { get; set; }
+            public string FieldComment { get; set; }
+
+            public List<PreviousManureApplicationYear> SelectPrevYrManureOptions { get; set; }
+            public string SelectPrevYrManureOption { get; set; }
+            public string PrevYearManureApplicationFrequency { get; set; }
+            public int? PrevYearManureApplicationNitrogenCredit { get; set; }
+
+            //public decimal? SoilTestNitrateOverrideNitrogenCredit { get; set; }
+            public string Placehldr { get; set; }
+
+            public bool IsSeasonalFeedingArea { get; set; }
+            public string SeasonalFeedingArea { get; set; }
+        }
+
+        public class CommandValidator : AbstractValidator<Command>
+        {
+            private readonly IOptions<AppSettings> _appSettings;
+
+            public CommandValidator(IOptions<AppSettings> appSettings)
+            {
+                _appSettings = appSettings;
+                RuleFor(x => x.FieldName).NotNull().WithMessage("Field Name is required");
+                RuleFor(x => x.FieldArea).NotNull().WithMessage("Field Area is required");
+                RuleFor(x => x.SelectPrevYrManureOption).NotEqual("select").WithMessage("Manure application in previous years must be selected");
+                RuleFor(x => x.FieldComment).MaximumLength(Convert.ToInt32(_appSettings.Value.CommentLength)).WithMessage("Exceeds maximum length of " + _appSettings.Value.CommentLength);
+            }
+        }
+
+        public class MappingProfile : Profile
+        {
+            public MappingProfile()
+            {
+                CreateMap<Field, Command>();
+            }
```

Comment:
You should start using the AutoMapper and save yourself the extra work.

CreateMap<Field, Command>()
   //Command as Destination
   .ForMember(m => m.FieldArea, opts => opts.MapFrom(s => s.area.ToString("G29")))
   .ForMember(m => m.FieldComment, opts => opts.MapFrom(s => s.comment))
   .ForMember(m => m.SelectPrevYrManureOption , opts => opts.MapFrom(s => s.prevYearManureApplicationFrequency))
.ReverseMap()
//FarmField as Destination
.ForMember(m => m.area, opts => opts.MapFrom(s => s..FieldArea != null ? Convert.ToDecimal(s.FieldArea) : 0))
.ForMember(m => m.SeasonalFeedingArea, opts => opts.MapFrom(s => s.IsSeasonalFeedingArea ? "Yes" : "No"))
;